docs/ref/migration-operations.txt

```diff
@@ -265,6 +273,10 @@ match the operation's place in the project history, and the second is an
 instance of :class:`SchemaEditor
 <django.db.backends.base.schema.BaseDatabaseSchemaEditor>`.
 
+The ``reverse_code`` argument is called when unapplying migrations. This
+callable is used so that the migration is reversible by undoing what is being
```

Comment:
This callable should undo what is done in the ``code`` callable so that the migration is reversible.